en_US/mqtt-over-quic/introduction.md

```diff
@@ -1,68 +1,149 @@
+   - It sends PUBLISH packets over the stream where it gets the topic subscription and also expects PUBACK for QoS1 from the same stream.
+
+
+::: tip
+
+The order of data is maintained per stream, hence, if there are two topics whose data is correlated and ordering is crucial, they should be mapped to the same stream.
+:::
 
-There are also more opportunities to be explored:
+#### Features and usecases:
+
+   - Decouple connection control and MQTT data exchange
+   
+     The MQTT connection control packets, such as CONNECT, CONNACK, and PING, are handled over the control stream, while data exchanges such as publishing and subscription are done over the data stream. Even if the data stream is slow, the connection can still be kept alive in terms of handling PINGREQ and PINGRESP.
+     
+   - Avoid head-of-line blocking issues among the topics
+   
+     MQTT-over-QUIC allows for multiple data streams for different topics, enabling the messages of different topics to be delivered independently.
+     
+   - Split uplink (publishing) and downlink (subscribe) data
+   
+     For example, a client can use one stream to publish QoS1 messages and handle PUBACK over that same stream, while receiving QoS0 messages from its subscriptions on another stream from the broker.
+     
+   - Prioritize different data
+     MQTT over QUIC also provides the ability to prioritize data from different MQTT topics through multi-streams. This means that topic data can be prioritized and delivered accordingly, improving the overall performance and responsiveness of the connection. 
+   
+   - Improves the parallelism of processing on the client/EMQX side.
+   
+     With the use of data streams, EMQX is able to process multiple streams in parallel, improving the overall efficiency and responsiveness of the system.
+     
+   - More robust handling of MQTT data
+     
+     If a single data stream is aborted due to an application error, it will not cause the entire connection to close. Instead, the application is free to recreate the stream and recover the data. This allows for more reliable and resilient MQTT communication.
+     
+   - Flow control data streams
+     
+     Flow control can be applied per data stream, allowing for different flow control policies for different topics or QoS levels.
+     
+   - Reduce subscription latency
+   
+     A client does not need to wait for MQTT CONNACK before sending the subscribe or publish packets.
+     
+     However, EMQX will only begin processing them after the client has established a connection and while the connection is allowed.
+     
+
+  Limitations:
+
+  - Preserving session state is currently not supported.
+  
+    This means that if a client needs to reconnect, it will have to resubscribe to the topics it was previously subscribed to over a data stream.
+    @TODO also describe the QoS message states
```

Comment:
QoS 1 and QoS2 message states are not preserved if the data stream is closed unexpectedly by either peer. also see `Future Work`